docs/cli/configuration.md

```diff
@@ -14,11 +14,13 @@ For example:
 ```bash
 $> raito <command> --config-file myconfig.yaml
 ```
-The configuration file allows you to specify multiple targets instead of only 1 (see later). Environment variables can be used in the configuration file by using double curly brackets, e.g. {% raw %}
+
+The configuration file allows you to specify multiple targets instead of only 1 (see later). Environment variables can be used in the configuration file by using double curly brackets, e.g. 
+
```

Comment:
The raw 'commands' that have been removed are required to render the double curly braces (they also have a meaning in markdown). Current you get this
![image](https://user-images.githubusercontent.com/31243536/215785636-12e14cbe-db2d-4cc1-b47f-b3ec63c2c471.png)